settings.gradle

```diff
@@ -18,7 +18,7 @@ pluginManagement {
             url = uri("https://maven.pkg.github.com/department-of-veterans-affairs/lighthouse-di-starter-boot")
             credentials(HttpHeaderCredentials) {
                 name = "Authorization"
-                value = "Bearer " + System.getenv("GITHUB_ACCESS_TOKEN")
+                value = "Bearer " + GITHUB_ACCESS_TOKEN
```

Comment:
From https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/d48fecf8a622c3a397f37ddaf9416237efc17f28#diff-7f825392aa37acd1cee0c2e7b9bb7366ad6eac64f3e6cdd816e156bcb69d30de